vm/src/obj/objset.rs

```diff
@@ -181,17 +181,18 @@ fn set_new(
 
 fn set_len(vm: &VirtualMachine, args: PyFuncArgs) -> PyResult {
     trace!("set.len called with: {:?}", args);
-    arg_check!(vm, args, required = [(s, Some(vm.ctx.set_type()))]);
+    arg_check!(vm, args, required = [(s, Some(vm.ctx.frozenset_type()))]);
```

Comment:
If changing the type here, also rename the function from `set_len` to `frozenset_len`.

Reply:
After changing `set` and `frozenset` to be separate types this will not look the same. Let's see where the discussion is going and then I will do fixes accordingly.